gen_cert.py

```diff
@@ -1752,7 +1752,7 @@ def _generate_v3_dynamic_certificate(
 
         # SECTION: Extra achievements
         achievements_string = ""
-        achievements_description_string = unicode(self.interstitial_texts[grade])
+        achievements_description_string = self.interstitial_texts[grade].decode('utf-8')
```

Comment:
not much difference BUT `.decode()` is available in python 3 and `unicode()` is not...